solutions/instances/variables.tf

```diff
@@ -214,6 +214,25 @@ variable "skip_scc_workload_protection_auth_policy" {
   description = "Set to true to skip the creation of an IAM authorization policy that permits the SCC instance created by this solution read access to the workload protection instance. Only used if `provision_scc_workload_protection` is set to true."
 }
 
+variable "attachments" {
```

Comment:
This input variable is too complex for a DA. It would be extremely hard for a user to figure out in the Projects UI how to structure this until they address this: https://github.ibm.com/ibmcloud/projects-ui/issues/1828

We have some options that maybe we should discuss..

a) Keep the current structure, but create some documentation that the variable description would link to on how the structure should be. Same way we did for KMS DA [here](https://github.com/terraform-ibm-modules/terraform-ibm-kms-all-inclusive/blob/main/solutions/standard/DA-keys.md).

b) Just take in a list of profiles and have the DA code itself automatically pick the latest ones and automatically scope it to the full account on a daily schedule.

c) Have a combination of both `a` and `b`; a simple list, and optional complex object with a corresponding doc for advanced customisation. 

Either way, I think we may want to have a default value of the latest fscloud profile. We could add `b` first and then eventually `a`? Any thoughts @vburckhardt

Reply:
@ocofaigh What do you feel about the approach I implemented where scope is optional, and if user doesn't want to specify it. Then it scopes to account.

Reply:
@jor2 Didn't we say to go with option b above for now? And automatically scope it to the full account. We can add a further custom variable with some documentation (option a) at a later point